File: Codigos estudiantes por lenguaje/PY/Bryann Valderrama/Grafos/matrizAdyacencia.py
from sys import stdin, stdout
rl = stdin.readline
wr = stdout.write

V, E = rl().strip().split()
V, E = int(V), int(E)

G = [[-1 for x in range(V)] for x in range(V)]
for i in range(V):
    G[i][i] = 0

for i in range(E):
    desde, hasta, peso = rl().strip().split()
    desde, hasta, peso = int(desde), int(hasta), int(peso)
    G[desde][hasta] = peso
    G[hasta][desde] = peso

for i in range(V):
    for j in range(V):
        if G[i][j] == -1:
            wr(f'inf\t')
        else:
            wr(f'{G[i][j]}\t')
    wr('\n')

File: Codigos estudiantes por lenguaje/C++/Bryann Valderrama/Grafos/MatrizAdyacencia.cpp
#include <cstdlib>
#include <bits/stdc++.h>
#define MAX_V 101
using namespace std;

int G[MAX_V][MAX_V];

int main(int argc, char const *argv[])
{
    int V, E;
    scanf("%i %i", &V, &E);
    G[V][V];
    for (int i = 0; i < V; i++)
    {
        memset(G, -1, sizeof(G));
    }
    for (int i = 0; i < V; i++)
    {
        G[i][i] = 0;
    }
    for (int i = 0; i < E; i++)
    {
        int desde, hasta, peso;
        scanf("%i %i %i", &desde, &hasta, &peso);
        G[desde][hasta] = peso;
        G[hasta][desde] = peso;
    }
    for (int i = 0; i < V; i++)
    {
        for (int j = 0; j < V; j++)
        {
            if (G[i][j] == -1)
            {
                cout << "inf \t";
            }
            else
            {
                cout << G[i][j] << "\t";
            }
        }
        cout << endl;
    }

    return 0;
}
